cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractStatefulModel.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.strimzi.api.kafka.model.status.Condition;
+import io.strimzi.api.kafka.model.storage.Storage;
+import io.strimzi.operator.common.Reconciliation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * AbstractModel an abstract base model for all components of the {@code Kafka} custom resource
+ */
```

Comment:
Javadoc seems to be from AbastractModel instead of being specific to this new abstraction for stateful resources.